controllers/past-grants/views/index.njk

```diff
@@ -83,45 +101,69 @@
-                                        'value': '0|10000'
-                                    }, {
-                                        'label': '£10,000–£50,000',
-                                        'value': '10000|50000'
-                                    }, {
-                                        'label': '£50,000-£100,000',
-                                        'value': '50000|100000'
-                                    }, {
-                                        'label': 'Over £100,000',
-                                        'value': '100000'
-                                    }],
-                                    silentlyOptional: true
-                                }) }}
-
-                                {% if queryParams.amount %}
-                                    <a class="search-filters__clear-facet" href="?{{ queryParams | removeQueryParam('amount') }}">
-                                        Clear amount
-                                    </a>
-                                {% endif %}
+
+                                <noscript>
+                                    {{ inputChoice({
+                                        type: 'radio',
+                                        name: 'amount',
+                                        label: 'Amount awarded',
+                                        value: queryParams.amount,
+                                        options: [{
+                                            'label': 'Any amount',
+                                            'value': ''
+                                        }, {
+                                            'label': 'Under £10,000',
+                                            'value': '0|10000'
+                                        }, {
+                                            'label': '£10,000–£50,000',
+                                            'value': '10000|50000'
+                                        }, {
+                                            'label': '£50,000-£100,000',
+                                            'value': '50000|100000'
+                                        }, {
+                                            'label': 'Over £100,000',
+                                            'value': '100000'
+                                        }],
+                                        silentlyOptional: true
+                                    }) }}
+
+                                    {% if queryParams.amount %}
+                                        <a class="search-filters__clear-facet" href="?{{ queryParams | removeQueryParam('amount') }}">
+                                            Clear amount
+                                        </a>
+                                    {% endif %}
+                                </noscript>
+
+                                <div class="js-only" v-cloak>
```

Comment:
We could probably create a view component (`input-choice`) to mirror the macro, and avoid needing to repeat form markup everywhere.

Reply:
Will do this when we're a bit further along with the UI – suspect it'll change a bit more before we're "done" so no point optimising too early.

Reply:
Aggreed